### PR TITLE
dts: msm8216-samsung: add e53g

### PR DESCRIPTION
**Pull request by gAtrium**
This commit adds E5 Duos (E500H) model node to the msm8216-samsung dts. Also lists the device under Documentation/devices.md.
Can't test if the panel works but another user reports that there's display already.

```
$ fastboot getvar all
....
(bootloader) lk2nd:model:Samsung Galaxy E5 (SM-E500H)
(bootloader) lk2nd:compatible:samsung,e35g

```

### CODE REVIEW

**Review by wonderfulShrineMaidenOfParadise — COMMENTED** [review]
Reword it to `dts: msm8216-samsung: add e53g`

**Comment by TravMurav** [comment]
Thanks!
